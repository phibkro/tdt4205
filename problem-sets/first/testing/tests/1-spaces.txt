go
go go
go gogo
 go go go  go go go 
gogo gogo
