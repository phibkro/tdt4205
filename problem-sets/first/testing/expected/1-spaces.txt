Table filled!
line    1: accepted: go
line    2: accepted: go go
line    3: error: go gogo
~~~~~~~~~~~~~~~~~~~~~~~^
line    4: accepted:  go go go  go go go 
line    5: error: gogo gogo
~~~~~~~~~~~~~~~~~~~~^
